### PR TITLE
[Fixbug] Two icons in gnome #813

### PR DESCRIPTION
**Pull request by tanrax**
Theoretically it shouldn't happen again... it's best to wait for feedback.

### CODE REVIEW

**Comment by laurent22** [comment]
So should it be merged?

**Comment by tanrax** [comment]
For my part, yes. It keeps working and maybe it will fix the duplicate icon problem.

File: install_ubuntu.sh [hunk 1]
```diff
@@ -16,13 +16,13 @@ rm -f ~/.joplin/*.AppImage ~/.local/share/applications/joplin.desktop
...
 # Download the latest version
-wget -O ~/.joplin/Joplin-$version-x86_64.AppImage https://github.com/laurent22/joplin/releases/download/v$version/Joplin-$version-x86_64.AppImage 
+wget -O ~/.joplin/Joplin.AppImage https://github.com/laurent22/joplin/releases/download/v$version/Joplin-$version-x86_64.AppImage 
 # Gives execution privileges
-chmod +x ~/.joplin/Joplin-$version-x86_64.AppImage
+chmod +x ~/.joplin/Joplin.AppImage
 # Download icon
 wget -O ~/.joplin/Icon512.png https://joplin.cozic.net/images/Icon512.png
 # Create icon for Gnome
-echo -e "[Desktop Entry]\nEncoding=UTF-8\nName=Joplin\nExec=/home/$USER/.joplin/Joplin-$version-x86_64.AppImage\nIcon=/home/$USER/.joplin/Icon512.png\nType=Application\nCategories=Application;" >> ~/.local/share/applications/joplin.desktop
```
Comment (tessus) [inline install_ubuntu.sh]: The error is here:

You are using `>>` which appends to the current file. This is wrong. you want to create a new file (replacing the old).

**Comment by tanrax** [comment]
Is it possible to deactivate the question? Otherwise you could delete the icon that is created. What is the path of the file?

**Comment by tessus** [comment]
@tanrax unfortunately I currently don't have a Linux box with a GUI to play around. I don't know which part of the code asks to create an icon. @laurent22 would know the details.

**Comment by tessus** [comment]
@tanrax btw, I also don't know the procedure for changes in PRs for this project. I usually like to squash them into one commit. I mean a PR can have several commits (when they are addressing separate things), but in this case I'd squash and force push.

**Comment by tanrax** [comment]
@tessus I don't have a problem :)

**Comment by laurent22** [comment]
Hi, not sure what's the status of this pull request? I've actually merged another related one [here](https://github.com/laurent22/joplin/pull/841). Does that other pull request have all the changes you need?

**Comment by tanrax** [comment]
It's being developed by another pull: https://github.com/laurent22/joplin/pull/833

**Comment by tanrax** [comment]
To fix in linux the integration of the icon I have seen the following configuration. Would it be possible to apply it?

https://github.com/electron-userland/electron-builder/issues/1962#issuecomment-454295183

**Comment by laurent22** [comment]
@tanrax, why would that fix the icon issue? This parameter seems to be only to ask or not ask for system integration?

**Comment by tanrax** [comment]
Screenshot from 2019-03-12 12-51-50
When it is updated it asks again for integration with the operating system, which causes the problem of the double icon. Searching I have found that it is possible to disable this option directly from electron.

**Pull request closed without merging**